app/SmartAlbums/Utils/MimicModel.php

```diff
@@ -29,6 +29,9 @@ abstract public function toArray();
 	 */
 	public function __get(mixed $key)
 	{
+		if (is_string($key)) {
```

Comment:
Don't you want... the negative? :laughing:

Reply:
so fast 😆 , just fixed it in #2707fde